src/StoreApi/Schemas/V1/CartSchema.php

```diff
@@ -390,6 +390,10 @@ protected function get_tax_lines( $cart ) {
 		$cart_tax_totals = $cart->get_tax_totals();
 		$tax_lines       = [];
 
+		if ( 'itemized' !== get_option( 'woocommerce_tax_total_display' ) ) {
```

Comment:
Minor nitpick, since we're returning early, can we move `$cart_tax_totals = $cart->get_tax_totals();` after the if, and save an unnecessary method call?